src/components/ContractsTable.vue

```diff
@@ -9,9 +9,9 @@
           </hint-tooltip>
         </th>
         <th>
-          Created By
+          Created
```

Comment:
Unrelated, I changed it from "Created By" back to "Created" because on current develop "Created By" header is duplicated in two columns.